backend/app/api/endpoints/wiki.py

```
@@ -26,53 +31,31 @@
-    # Second, try user JWT token (recommended method)
+    # A user JWT is available to the executor through TASK_INFO.
     try:
-        # Verify JWT token and get user
         user = security.get_current_user_from_token(token, db)
         if user and user.is_active:
-            logger.debug(
-                f"Wiki content write authenticated via JWT token for user {user.id}"
-            )
-            return
+            return user
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)